resources/services/elasticbeanstalk/environments.go

```diff
@@ -462,6 +481,9 @@ func fetchElasticbeanstalkConfigurationOptions(ctx context.Context, meta schema.
 		options.Region = c.Region
 	})
 	if err != nil {
+		if IsInvalidParameterValueError(err) {
+			return nil
```

Comment:
debug log please

Reply:
Great point! Added in https://github.com/cloudquery/cq-provider-aws/pull/751/files#diff-e4759eab9747ef3abc27af2730ff5e7c5e63f486650f5678f85b8441a8db3a52R459

Reply:
Done!